.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint", "eslint-plugin-tsdoc", "import", "html"],
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:import/errors",
+      "plugin:import/warnings"
+    ],
+    "rules": {
+      // disable the rule for all files]
+      "@typescript-eslint/explicit-function-return-type": "off",
+      "@typescript-eslint/no-non-null-assertion": "off",
+      "import/named": "off",
+      "import/no-unresolved": "off",
+      "import/extensions": ["error", "always", { "ignorePackages": true }],
+      "import/no-duplicates": "off",
+      "no-duplicate-imports": "off",
+      "tsdoc/syntax": "warn"
+    }
+  }
```

Comment:
Which of the above excludes this `No newline at end of file` rule. I would expect the automated test on github actions to fail otherwise.